ArjunaJTS/standalone/start.sh

```diff
@@ -1,6 +1,6 @@
 
 # JBoss, Home of Professional Open Source
-# Copyright 2016, Red Hat, Inc., and others contributors as indicated
+# Copyright 2020, Red Hat, Inc., and others contributors as indicated
```

Comment:
```suggestion
# Copyright 2016, Red Hat, Inc., and others contributors as indicated
```

Reply:
Thanks for your feedback, I've just removed all date changes from the commit.